src/Tizen.Diagnostics/Tizen.Diagnostics/ReportReceivedEventArgs.cs

```diff
@@ -0,0 +1,40 @@
+/*
+* Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using NativeApi = Interop.Diagnostics;
+
+namespace Tizen.Diagnostics
+{
+    /// <summary>
+    /// ReportReceivedEventArgs is an extended EventArgs class.
+    /// This class contains event arguments for the ReportReceived event from the Diagnostics class.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public class ReportReceivedEventArgs : EventArgs
+    {
+        internal ReportReceivedEventArgs(IntPtr ctx)
+        {
+            Context = new DiagnosticsContext(ctx);
+        }
```

Comment:
I think, it is better reasonable, ReportReceivedEventArgs class does not need to know creation of context.
```suggestion
        internal ReportReceivedEventArgs(DiagnosticsContext ctx)
        {
            Context = ctx;
        }
```